stdlib/Bool.dora

```diff
@@ -1,5 +1,8 @@
 internal class Bool {
   internal fun toInt() -> Int;
+
+  fun hash() -> Int = self.toInt();
```

Comment:
Do we want `Long` as type for the hash?

Reply:
I have considered it, but I think if someone has problems with hashcode collisions with a space of 4 billion different values, another 32 bit won't save that person.